src/request/src/network.js

```diff
@@ -394,76 +394,75 @@ export class KinveyRequest extends NetworkRequest {
       })
       .catch((error) => {
         if (error instanceof InvalidCredentialsError && retry === true) {
-          return LocalRequest.getActiveUser(this.client)
-            .then((activeUser) => {
-              if (!isDefined(activeUser)) {
-                throw error;
-              }
-
-              const socialIdentities = activeUser._socialIdentity;
-              const sessionKey = Object.keys(socialIdentities)
-                .find(sessionKey => socialIdentities[sessionKey].identity === SocialIdentity.MobileIdentityConnect);
-              const session = socialIdentities[sessionKey];
-
-              if (isDefined(session)) {
-                // Refresh MIC Token
-                if (session.identity === SocialIdentity.MobileIdentityConnect) {
-                  const refreshMICRequest = new KinveyRequest({
+          const activeUser = LocalRequest.getActiveUser(this.client);
```

Comment:
@vinaygahlawat This line is the main change that is needed. You will notice it was changed from expecting a `Promise` to be returned to just the `activeUser`.